src/components/shared/formItems/ProjectPayoutMods.tsx

```diff
@@ -419,14 +417,13 @@ export default function ProjectPayoutMods({
             />
           ) : (
             <FormItems.ProjectHandle
-              name="project"
+              name="handle"
```

Comment:
Setting the `name` to `handle` to match the form state variable ensures that the `Form.Item` binds the `input` to the `form` state correctly.